internal/cli/user_modify.go

```diff
@@ -50,11 +50,20 @@ func (c *UserModifyCommand) Run(args []string) int {
 	user := userResp.User
 
 	// Perform modifications
+	willModify := false
 	if v := c.flagNewUsername; v != "" {
 		user.Username = v
+		willModify = true
 	}
 	if v := c.flagDisplay; v != "" {
 		user.Display = v
+		willModify = true
+	}
+
+	if !willModify {
+		c.ui.Output("At least one user modification flag must be specified...\n\n"+
```

Comment:
```suggestion
		c.ui.Output("at least one user modification flag must be specified"+
```

Nit: After a quick check of `internal/cli/` I believe "most" of our CLI errors have a lowercase error message and don't have the `...` or new lines, so I suggest we modify this to be consistent.